### PR TITLE
Add keybind for showing profile

### LINKED ISSUE #9941: Add "open profile" hotkey
**Describe the new feature:** 
Ctrl + F (maybe another). 
The first press opens the status bar, and the second press will open the profile itself. 
If you are without a login, then let it open the login window.

### PR DESCRIPTION
**Pull request by andrewhong04**
Fixes: https://github.com/ppy/osu/issues/9941

### CODE REVIEW

File: osu.Game/Localisation/GlobalActionKeyBindingStrings.cs [hunk 1]
```diff
@@ -149,6 +149,11 @@ public static class GlobalActionKeyBindingStrings
         /// </summary>
         public static LocalisableString ToggleNotifications => new TranslatableString(getKey(@"toggle_notifications"), @"Toggle notifications");
 
+        /// <summary>
+        /// "Show Profile"
+        /// </summary>
+        public static LocalisableString ShowProfile => new TranslatableString(getKey(@"toggle_notifications"), @"Show Profile");
```
Comment (smoogipoo) [inline osu.Game/Localisation/GlobalActionKeyBindingStrings.cs:155]:
```suggestion
        public static LocalisableString ShowProfile => new TranslatableString(getKey(@"show_profile"), @"Show profile");
```

File: osu.Game/Localisation/GlobalActionKeyBindingStrings.cs [hunk 2]
```diff
@@ -149,6 +149,11 @@ public static class GlobalActionKeyBindingStrings
         /// </summary>
         public static LocalisableString ToggleNotifications => new TranslatableString(getKey(@"toggle_notifications"), @"Toggle notifications");
 
+        /// <summary>
+        /// "Show Profile"
```
Comment (smoogipoo) [inline osu.Game/Localisation/GlobalActionKeyBindingStrings.cs:153]:
```suggestion
        /// "Show profile"
```

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game/Localisation/GlobalActionKeyBindingStrings.cs [hunk 3]
```diff
@@ -149,6 +149,11 @@ public static class GlobalActionKeyBindingStrings
         /// </summary>
         public static LocalisableString ToggleNotifications => new TranslatableString(getKey(@"toggle_notifications"), @"Toggle notifications");
 
+        /// <summary>
+        /// "Show profile"
+        /// </summary>
+        public static LocalisableString ShowProfile => new TranslatableString(getKey(@"toggle_notifications"), @"Show profile");
```
Comment (smoogipoo) [inline osu.Game/Localisation/GlobalActionKeyBindingStrings.cs:155]:
```suggestion
        public static LocalisableString ShowProfile => new TranslatableString(getKey(@"show_profile"), @"Show profile");
```

**Review by smoogipoo — CHANGES_REQUESTED** [review]

File: osu.Game/Localisation/GlobalActionKeyBindingStrings.cs (hunk 3 above)
Comment (novialriptide) [reply osu.Game/Localisation/GlobalActionKeyBindingStrings.cs:155]: That was silly of me.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by smoogipoo — DISMISSED** [review]

**Comment by peppy** [comment]
Feels a bit weird that this isn't a toggle, but maybe okay?

**Comment by peppy** [comment]
@bdach can i get an opinion from you on this being a non-toggle? can't decide which feels better. in fact i'm almost leaning to making all overlays non-toggle and have people use escape to close, but not sure.

maybe best to make this PR be a toggle and discuss the UX later?

**Comment by novialriptide** [comment]
> maybe best to make this PR be a toggle and discuss the UX later?

@peppy Done.

**Comment by smoogipoo** [comment]
@peppy wants the entire overlay to toggle, not just the text to be changed.

**Comment by novialriptide** [comment]
I'm not too keen on this, because it doesn't actually bring up the user profile overlay.

**Comment by bdach** [comment]
> @bdach can i get an opinion from you on this being a non-toggle? can't decide which feels better.

I don't have a strong opinion to give. I will say that I've gotten quite used to these being toggles - escape is not always a viable and reliable alternative, as you may need to press it however many times if there are nested focused elements in the overlay. I also think say that this PR should probably be consistent with everything else, i.e. it should toggle, and if toggle is deemed bad, then all of the overlays should be changed to not toggle in one PR.

File: osu.Game/Overlays/UserProfileOverlay.cs [hunk 4]
```diff
@@ -41,6 +42,18 @@ public UserProfileOverlay()
...
+        {
+            switch (e.Action)
+            {
+                case GlobalAction.ToggleProfile:
+                    Hide();
+                    return true;
+            }
+
+            return base.OnPressed(e);
+        }
```
Comment (peppy) [inline osu.Game/Overlays/UserProfileOverlay.cs:55]: Please move this to `OsuGame` to keep everything in one place. This kind of cross-dependency makes understanding and ensuring correctness very hard.

You can call `userProfile.Hide()` in the case it is already shown.

**Review by peppy — CHANGES_REQUESTED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**